torch/multiprocessing/reductions.py

```diff
@@ -177,8 +177,8 @@ def rebuild_cuda_tensor(
             storage_cls, (storage_handle, storage_offset_bytes)
         )
         if storage is None:
-            torch.cuda._lazy_init()
-            storage = storage_cls._new_shared_cuda(
+            torch.get_device_module()._lazy_init()
```

Comment:
Don't you want to specifically initialize `storage_device`, instead of "the device module"?

Reply:
@ezyang Yes, I use get_device_module to lazy_init only for a specific device. Previously it was only cuda, now it is replaced with the third-party device for inference.